backend/src/common/enums/db/foreing-keys.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum ForeingKeys {
```

Comment:
```suggestion
enum ForeignKey {
```
Enum name should be singular